rj_gameplay/rj_gameplay/action/capture.py

```diff
@@ -22,11 +21,11 @@ def tick(self, intent) -> None:
         collect_command = CollectMotionCommand()
         intent.motion_command.collect_command = [collect_command] 
         intent.dribbler_speed = 1.0
-        intent.active = True
+        intent.is_active = True
 
         return intent
 
     def is_done(self, world_state) -> bool:
-        if world_state.our_robots[self.robot_id].has_ball_sense:
+        if not self.robot_id is None and world_state.our_robots[self.robot_id].has_ball_sense:
```

Comment:
When will `robot_id` be `None`?

Reply:
when capture action is initialized we have to default all to None